src/screens/tools/Charts.js

```diff
@@ -1,30 +1,23 @@
 import React, { Suspense } from "react";
+import { Switch, Route, useRouteMatch } from "react-router-dom";
 
-const ChartView = () => {
-  return (
-    <div className="tabs is-toggle is-toggle-rounded">
-      <ul>
-        <li className="is-active">
-          <a href="/charts">
-            <span>Distribution</span>
-          </a>
-        </li>
-        <li>
-          <a href="/charts">
-            <span>Hash Rate</span>
-          </a>
-        </li>
-      </ul>
-    </div>
-  );
-};
+// Containers
+import DailyDifficulty from "containers/charts/DailyDifficulty";
+
+//@todo we should move all charts into components err containers @todo this today before merging..
+//@todo we need to figure out a better color scheme here as well.
+//@todo would be cool to link "source" like Etherscan does, but link it to our API docs when we have them.
+//@todo right now we default to difficulty for the "url"/charts -> I think we can in the future make this a lander for charts, rather than just default to difficulty similar to etherscan. Allow people to pick it out..
```

Comment:
I like that idea for when we have more chart data.

- Reference: https://etherscan.io/charts